tests/kafkatest/services/verifiable_consumer.py

```diff
@@ -48,19 +50,27 @@ def handle_shutdown_complete(self):
         self.assignment = []
         self.position = {}
 
-    def handle_offsets_committed(self, event):
+    def handle_startup_complete(self):
+        self.state = ConsumerState.Started
+
+    def handle_offsets_committed(self, event, node, logger):
         if event["success"]:
             for offset_commit in event["offsets"]:
+                if "error" in offset_commit and offset_commit["error"] != "":
+                    logger.warn("%s: Offset commit failed for: %s" % (str(node.account), offset_commit))
```

Comment:
failing offset commits leads to later problems, so it is valuable to see this one. Changed it to debug though